LocalMultiplayerAgent/DeploymentTool/readme.md

```diff
@@ -0,0 +1,57 @@
+﻿After testing their game server(s) on LMA, you may want to go ahead and deploy them.
+This tool was developed exactly for this purpose; allow you go ahead to deploy your game servers.
```

Comment:
as I mentioned above, we can add more explanation here - where are the game servers deployed? how do we verify it after running the script? what are the steps?